packages/remix/src/ssr/authenticateRequest.ts

```diff
@@ -6,13 +6,19 @@ import { getEnvVariable } from '../utils';
 import type { LoaderFunctionArgs, RootAuthLoaderOptions, RootAuthLoaderOptionsWithExperimental } from './types';
 import { parseCookies } from './utils';
 
+const handleIsSatelliteBooleanOrFn = (opts: RootAuthLoaderOptionsWithExperimental, url: URL) => {
```

Comment:
@panteliselef this is a good candidate to move into the `utils` file in the same directory

Reply:
@raptisj Yes you are right. Although this is on my list for making it a utility available across packages as we handle similar logic in clerk-js, remix, nextjs.